Translations-Recommendations-Guide.md

```diff
@@ -0,0 +1,99 @@
+
+
+# 4. Technical procedure to create a translated file
+
+* Install git in your computer and create an account in github.com.
+* From your github.com account *fork* the FreeCodeCamp repository (https://github.com/FreeCodeCamp/FreeCodeCamp). A directory for the forked repository should appear in your account.
+* Clone your fork into your computer as in the following example (replace the word myaccount with your github account's name  [find some help here](https://help.github.com/articles/fork-a-repo/)):
+```sh
+git clone --depth=1 https://github.com/myaccount/FreeCodeCamp.git
+cd FreeCodeCamp
+```
+*  Now, on GitHub, go to the *issues* section of the official repository and search for the language you wanna work on (i.e https://github.com/FreeCodeCamp/FreeCodeCamp/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Atranslation). Find an issue that nobody is working on -or that has been left unattended during 2 weeks or more (ask first in the chat room)-.  You can take it by simply adding a comment, such as “*Working on it*”.
+*  In your forked repository, create a *branch* with a name such as “add/LG-translation-XXX” donde LG is the abbreviated name of the language you're working on, and XXX is the name of the file you're going to modify.  ([more help here](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches)):
+```sh
+git checkout -b add/ES-translation-basic-javascript
+```
+* Find the JSON file in the ```seeds/challenges``` directory in your local copy. Find the challenge of your interest and modify it by adding the name of the challenge in ```"nameLG"``` and its description in ```"descriptionLG"```, where "LG" is the language of your choice. It's important to keep the proper syntax of the JSON file. You can verify this by installing ```node.js``` and passing the JSON file to the interpreter. For example: ```node seeds/challenges/basic-javascript.json```. You will also be able to verify the syntax by copying and pasting the content of the file in http://jsonlint.com. If you think that you can't continue for a while with the translation, you can leave a comment on the same issue, indicating that another person should continue. You can push your changes so far so the next person won't duplicate your hard work.
+* When you're done, *commit* your changes with a brief description in English (regardless of the language). As a part of the message of the commit, and after the description, add "Closes #yyyy" where yyyy is the number of the issue you're working on (this is convenient because it closes the issue automatically when your change gets merged with the main repository.). Afterwards, *push* the changes to your fork (see [the help](https://help.github.com/articles/pushing-to-a-remote/)):
+```sh
+git commit -m "Challenges of Basic Javascript translated. Closes #4234" seed/challenges/basic-javascript.json 
```

Comment:
Todo: 
As per the current guidelines, we are NOT supposed to add issue no's in the commit message, rather this should be in the PR description?

This needs revisiting in the original guides in Spanish aswell.